index.js

```diff
@@ -173,6 +179,135 @@ app.get("/continentFilter", function (req, res, next) {
 
 });
 
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/cart', function (req, res) {
+    var cartId = req.body.cartId;
+    var itemId = req.body.item._id;
+    var quantity = req.body.quantity;
+    var imageUrl = req.body.imageUrl;
+
+
+    if(cartId === 'undefined') {
```

Comment:
How is cartId passed from the UI? In a nice world it shouldn't be string compared.

Reply:
Still 'undefined' is checked